suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -598,3 +596,27 @@ export const onPassphraseSubmit = createThunk(
         });
     },
 );
+
+type DeviceConnectThunkEventType = typeof DEVICE.CONNECT | typeof DEVICE.CONNECT_UNACQUIRED;
+
+const deviceConnectThunksMap: Record<
+    DeviceConnectThunkEventType,
+    (payload: DeviceConnectActionPayload) => any
+> = {
+    [DEVICE.CONNECT]: deviceActions.connectDevice,
+    [DEVICE.CONNECT_UNACQUIRED]: deviceActions.connectDevice,
+};
+
+type DeviceConnectThunksParams = {
+    type: DeviceConnectThunkEventType;
```

Comment:
I believe `type` name is confusing here. How about `connectThunkEventType` or something like that?

Reply:
Cant change it it is a type from action (`action.type`), the result is an action